packages/react/src/index.ts

```diff
@@ -57,7 +57,6 @@ export type {
   PromiseResult,
   Result,
   Success,
-  Url,
```

Comment:
Verify if export `URI` and `URL` is needed here or already done elsewhere.

Reply:
they are now only used internally in our packages, don't need to export them at all